.github/workflows/snapshot-tests.yml

```diff
@@ -30,18 +30,22 @@ jobs:
         with:
           python-version: 3.12
 
```

Comment:
@prabhu Not sure how else to get a working installation of the targeted branch. I tried doing corepack but it refused to work and it seems like other people just use the action. At first I just put version 9 for pnpm action, but because the package.json specifies 9.2.0, it did not work.

However, I will add a step to grab the pnpm version in the package.json first, then use it for the action.

Reply:
@prabhu Apparently the action doesn't actually require a version to be specified and will look at the package.json. So that's good!